.gitignore

```diff
@@ -16,3 +16,4 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+.DS_Store
```

Comment:
Nice. I normally just flag OSX to stop producing them but I just upgraded to Sierra which defaulted that setting. Fail.